uvicgse/project-virtual-team-2
# UPDATED: Partially implement issue #84: Improve stash branch

This feature allows the user to create a new branch from the commit of a stash and then applies the stash on top of the new branch. This is done by opening the `stash-modal` from the nav-bar, clicking on the dropdown menu of a particular stash, and clicking on the branch button. This will open the `branch modal` where you can name, create, and checkout the branch. The user can also open the `branch-modal` and create a new branch from the most recent stash at index 0.

This PR also recolors the stash nodes in the graph to the same orange as the modified files in the `file-panel` to indicate that it is a stash and not a commit.

To test this feature: 
- have or create a stash
- create a branch from the stash as described above
- confirm its success by examining the branch menu,  branch tags in the graph, and the terminal and log for accuracy. 
- confirm that the stash files were re-applied properly

**Update**: This branch is no longer merged with master and exists in a working state. Please review but do not merge until master is reverted _Need update from other group to see if this was fixed._
